src/settings/settings.ts

```diff
@@ -208,6 +208,39 @@ export default class SettingTab extends PluginSettingTab {
                     await this.plugin.saveSettings();
                 });
             });
+        new Setting(containerEl)
```

Comment:
See above. If we make this an ignore list, it should be styled similarly to dice formulas, probably.

We can also add in a folder selector.

Let me know if you would like me to do this.

Reply:
I'll see if i have time tomorrow.